tests/testthat/test-01-data-retrieval.R

```diff
@@ -103,11 +103,11 @@ test_that("default data path functions work if unset", {
   expect_error(use_default_data_path())
 
   data_path <- tempdir()
-  expect_error(m <- capture_output(use_default_data_path(data_path)), NA)
-  expect_match(m, "Call `usethis::edit_r_environ\\(\\)` to open '.Renviron'")
-  expect_match(m, "Store your data path with a line like:")
-  expect_match(m, "PORTALR_DATA_PATH=")
-  expect_match(m, "Make sure '.Renviron' ends with a newline!")
+  expect_error(m <- capture_messages(use_default_data_path(data_path)), NA)
+  expect_match(m, "Call \033\\[90m`usethis::edit_r_environ\\(\\)`\033\\[39m to open \033\\[34m'.Renviron'\033\\[39m", all = FALSE)
```

Comment:
Hmm, these ANSI escapes shouldn't be here.

Arguably they should be prevented at the source in this context.

But it's safer and easy enough to just make sure that crayon is disabled when you run your tests. I recommend you do something like I do in `setup.R`:

https://github.com/r-lib/usethis/blob/218a53cc8ab45a3e41a723e1283aa7ddd4e3c4aa/tests/testthat/setup.R#L11-L16

Reply:
FWIW, separate from the crayon thing, I often use `fixed = TRUE` in `expect_match()`, if I can get away with it, so I don't have to escape so many things.

Reply:
AHHHHHH, this would have made life a lot easier... 🙀  Thanks!